nextflow.config

```diff
@@ -73,8 +69,8 @@ params {
 
     // Schema validation default options
     validationFailUnrecognisedParams = false
-    validationLenientMode            = false
-    validationSchemaIgnoreParams     = 'genomes,igenomes_base,monochromeLogs'
+    validationLenientMode            = true
```

Comment:
Why true?

Reply:
That is a leftover from fixing the other bug with the string/integer

Reply:
Perhaps we revert it back since it's no longer needed.
```suggestion
    validationLenientMode            = false
```